src/components/TextArea/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef } from 'react';
+import propTypes from 'prop-types';
+
+import './style.scss';
+
+const TextArea = forwardRef(({
+  placeholder = '',
+  id,
+  type = 'text',
+  rows = 7,
+  cols = 50,
+  ...props
+}, ref) => (
+  <textarea
+    ref={ref}
+    className="retro-input"
+    placeholder={placeholder}
+    id={id}
+    type={type}
+    rows={rows}
+    cols={cols}
+    {...props}
+  />
+));
+
+TextArea.propTypes = {
+  placeholder: propTypes.string,
+  id: propTypes.string.isRequired,
+  type: propTypes.string,
+  rows: propTypes.oneOfType([
+    propTypes.string,
+    propTypes.number,
+  ]),
+  cols: propTypes.oneOfType([
+    propTypes.string,
+    propTypes.number,
+  ]),
```

Comment:
Why can it be both? Shouldn't it be just one?

Reply:
![image](https://github.com/lucianocaus/NeoPosts-React/assets/79598551/10fa40ce-8a19-4764-80d4-992dec13f484)

One of them was unnecessary